pkg/apis/topology/v1alpha2/helper/numanode/numanode.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package numanode
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	canonicalName = "node"
+	canonicalSep  = "-"
+)
+
+// IDToName creates the canonical NUMA node name given the NUMA cell ID
+func IDToName(numaID int) (string, error) {
+	if numaID < 0 {
+		return "", fmt.Errorf("invalid NUMA id range numaID: %d", numaID)
+	}
+	return canonicalName + canonicalSep + strconv.Itoa(numaID), nil
+}
+
+// NameToID retrieves the NUMA cell ID (integer) from the canonical
+// NUMA node name (node-X, cell ID would be X).
+func NameToID(node string) (int, error) {
+	splitted := strings.Split(node, canonicalSep)
+	if len(splitted) != 2 {
+		return -1, fmt.Errorf("invalid zone format zone: %s", node)
```

Comment:
`invalid zone format zone` doesn't sound right 😄 
maybe `invalid zone format: %d` would be enough?

Reply:
right, this is a relic from the original code coming from `k-sigs/scheduler-plugins`. I'll review and fix. Thanks for raising!

Reply:
Yeah, the wording of the error messages look a bit wonky. Other than that 👍